src/collinearity_tool/collinearity_tool.py

```diff
@@ -69,8 +76,50 @@ def vif_bar_plot(x, y, df, thresh):
     Examples
     --------
     >>> from collinearity_tool.collinearity_tool vif_bar_plot
-    >>> vif_bar_plot("response", ["exp1", "exp2", "exp3"], data, 5)
+    >>> vif_bar_plot(["exp1", "exp2", "exp3"], "response", data, 5)
     """
+    if type(x) is not list:
+        raise ValueError("x must be a list of explanatory variables!")
+    if type(y) is not str:
+        raise ValueError("y must be a string!")
+    if type(df) is not pd.DataFrame:
+        raise ValueError("df must be a pandas data frame!")
+    if type(thresh) is not int and type(thresh) is not float:
+        raise ValueError("thresh must be an integer or a float!")
+    
+    # Data frame containing VIF scores
+    explanatory_var = "+".join(set(x))
+    
+    y, X = dmatrices(y + " ~" + explanatory_var, df, return_type = "dataframe")
+    
+    vif_list = []
+    for i in range(X.shape[1]):
+        vif_list.append(variance_inflation_factor(X.values, i))
+        
+    vif_df = pd.DataFrame(vif_list, 
+                          columns = ["vif_score"])
+    vif_df["explanatory_var"] = X.columns
+    
```

Comment:
I would recommend removing the intercepts. Looking at similar bar charts online, it does not seem typical to include them. Examples ([one](https://elifesciences.org/articles/53036/figures), [two](https://www.statology.org/variance-inflation-factor-r/). It just seems there is a risk that the important information will be lost.

```suggestion
if intercept is False:
        vif_df = pd.DataFrame(
            vif_df[(vif_df.explanatory_var != 'Intercept')])
```